packages/react-components/react-table/src/contexts/dataGridContext.ts

```diff
@@ -11,6 +12,7 @@ const dataGridContextDefaultValue: DataGridContextValue = {
   selectableRows: false,
   selectionAppearance: 'brand',
   focusMode: 'none',
+  compositeRowTabsterAttribute: {} as TabsterDOMAttribute,
```

Comment:
Why not just `tabsterAttributes`?

Reply:
because it's only for:
* row
* composite navigation pattern

Reply:
If we needed other tabster attributes for other keyboard navigation patterns - it would be a good idea to clearly differentiate between them IMO.

`tabsterAttributes` sounds like it could be applied anywhere. in the component tree